ReactSkia/components/RSkComponentImage.cpp

```diff
@@ -20,7 +20,10 @@
 #include "ReactSkia/sdk/CurlNetworking.h"
 #include "ReactSkia/views/common/RSkImageUtils.h"
 #include "ReactSkia/views/common/RSkConversion.h"
+#include<mutex>
 
+#define MILLSEC_CONVERTER(time) time*1000
```

Comment:
Can we rename this to reflect from which unit to ms . we are converting. It will be good to have this function as part of our utils, considering it can be reused.